test/chpldoc/compflags/comment/badClose.doc.bad

```diff
@@ -1,4 +1,5 @@
-warning: chpldoc comment not closed, ignoring comment: /** in Line no: 1
+warning: chpldoc comment not closed, ignoring comment: * This comment is not closed properly
+Line no: 1
```

Comment:
Two things:
1) Do you really want to be concatenating the comment style characters after the opening `/*` into the comment that is output?
2) The result of this PR should be a clean match against the .good file for this test, meaning that the .bad file is no longer necessary and that it and the .future can be removed